### PR TITLE
Link to gps's mozilla-version-control-tools readthedocs for mercurial: h...

### PR DESCRIPTION
**Pull request by vaibhavmagarwal**
...ttp://mozilla-version-control-tools.readthedocs.org/en/latest/hgmozilla/

### CODE REVIEW

File: index.rst [hunk 1]
```diff
@@ -1,5 +1,6 @@
 A-Team Bootcamp
 ===============
+ .. image:: https://wiki.mozilla.org/images/thumb/1/13/Ateam.png/173px-Ateam.png
```
Comment (wlach) [inline index.rst]: This is probably a worthwhile change also, but could you put it in a separate commit / PR?

**Comment by vaibhavmagarwal** [comment]
Done. This now contains just the link to gps blog post. I will send another pr for the ateam logo.

**Comment by wlach** [comment]
Perfect, thank you!  :)
